client/editor.html

```diff
@@ -191,5 +162,39 @@
 			ArrayTileInfo = jqxhr.responseJSON;
 		});
 	</script>
+
+	<script>
+
+	window.onload = function() {
+	    console.log(ArrayTileInfo);
+	    var loc = location.search;
+	    var vars = loc.split("=");
+	    leveljson = "";
+	    if (vars.length > 1) {
+	        leveljson = "tracks/" + vars[1];
+	    }
+	    jaws.assets.root = "assets/";
+
+	    m_listImgURL = [];
+
+	    for (var i = 0; i < ArrayTileInfo.length; i++) {
```

Comment:
I didn't do this code, you did, I just moved it
But i don't think there is ajax here